pkg/fake/dynamic_client.go

```diff
@@ -3,7 +3,7 @@ package fake
 import (
 	"time"
 
-	. "github.com/onsi/gomega"
+	g "github.com/onsi/gomega"
```

Comment:
In general dot import is discouraged but an exception is made for ginkgo and gomega otherwise it kind of detracts from the fluency of BDD. Is there anyway we can exclude dot import?

Reply:
Yeah, I was torn about this one too. I also like the simplicity of the directly imported functions, but as I thought about it more it didn't seem to bad to have to qualify them like normal objects.

There are some other ways of avoiding dot imports described here, but they are a bit more complex and they also kinda-endorse this more verbose method.

https://onsi.github.io/ginkgo/#avoiding-dot-imports

Reply:
yeah creating top-level `vars` to effect dot import seems a bit overboard to me just to explicitly avoid dot import. I still prefer dot import in these 2 cases - both for the fluency aesthetics and ease of auto-complete in the IDE. In other code I've seen using gingko, it does seem tom be the norm.

Reply:
Per discussion on Automation call we're waiting on @skitt and @mangelajo to get back from PTO before making this decision, as we suspect they have thoughts and if so they should definitely be heard first.

Reply:
Idiomatic Ginkgo relies on the dot-imports, I think it would be a shame to lose that...

Reply:
Okay, so I'll propose an update with an exception to ignore Ginkgo and Gomega dot imports.